src/Modules/Finance/Header/Header.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import { TargetSum } from './TargetSum/TargetSum';
+import { Total } from './Total/Total';
+import { Transaction } from './Transaction/Transaction';
+
+import styles from './Header.module.css';
+
+type IHeaderProps = {
+
+}
+
+type IHeaderState = {
+	targetSumValue: string | number | any,
+	targetSumConfirmed: boolean,
+	totalSum: string,

```

Comment:
The names of both `totalSum` and `currentTransactionValue` imply that they're gonna be numbers, don't they?

Reply:
Yes, they do. This is my temporary decision caused by actual type of data I get from user's input.

Reply:
will be converted in a bug later